src/actions/make/_make.lua

```diff
@@ -28,9 +28,9 @@
 			dotnet = { "mono", "msnet", "pnet" }
 		},
 
-		onSolution = function(sln)
+		onWorkspace = function(wrk)
```

Comment:
I fear there may be gravity in this line. There is a tendency to propagate conventions like this...

I'm not sure I like `wrk` here. I'm not sure why, but it makes me feel awkward :)
Considering it's going to appear hundreds or thousands of times throughout premake...
Alternatives that come to mind are `wks` or `wsp`. Perhaps we can get a sample on preference?

I know this is a massive bikeshed (sorry!), But it's best to make a call on it before it proliferates.